virttest/libvirt_vm.py

```diff
@@ -1435,17 +1436,46 @@ def add_tpmdevice(help_text, device_path, model=None, type=None):
             else:
                 virt_install_cmd += " --boot emulator=%s" % emulator_path
 
+        # TODO: The current code uses a method that specifies a particular
+        # loader file to boot the guest. However, the latest virt-install
+        # supports using a method like "--boot uefi" to automatically match
+        # the ovmf file. In the future, when time permits, an interface could
+        # be abstracted to support this usage.
         bios_path = params.get("bios_path", None)
         if bios_path:
             if not has_sub_option("boot", "loader"):
-                LOG.warning("bios option not supported by virt-install")
+                LOG.warning(
+                    "'loader' option is not supported by this version of virt-install"
+                )
             else:
                 if "--boot" in virt_install_cmd:
                     virt_install_cmd += ","
                 else:
                     virt_install_cmd += " --boot "
                 virt_install_cmd += "loader=%s" % bios_path
 
+                bios_props = ast.literal_eval(params.get("bios_props", "{}"))
+                for sub_option, value in bios_props.items():
+                    if value is not None:
+                        if has_sub_option("boot", sub_option):
+                            virt_install_cmd += ",%s=%s" % (sub_option, value)
+                        else:
+                            LOG.warning(
+                                "'%s': option not supported by virt-install"
+                                % sub_option
+                            )
+
+                bios_vars_template_path = params.get("bios_vars_template_path", "")
```

Comment:
Same as the above, pls add the docstring for the parameter `bios_vars_template_path`, thanks.

Reply:
Done.